pdg/src/snapshots/c2rust_pdg__tests__analysis_test_pdg_snapshot.snap

```diff
@@ -687,6 +687,22 @@ g {
 }
 nodes_that_need_write = []
 
+g {
+	n[0]:  alloc      _     => _2  @ bb1[2]:  fn test_ref_field; _2 = pointers::calloc(move _3, move _4)  
+	n[1]:  copy       n[0]  => _1  @ bb2[3]:  fn test_ref_field; _1 = move _2 as *mut pointers::S (Misc)  
+	n[2]:  copy       n[1]  => _6  @ bb2[6]:  fn test_ref_field; _6 = &mut (*_1)                          
+	n[3]:  field.3    n[2]  => _   @ bb2[9]:  fn test_ref_field; _7 = (((*_6).3: pointers::T).3: i32)     
+	n[4]:  field.3    n[3]  => _7  @ bb2[9]:  fn test_ref_field; _7 = (((*_6).3: pointers::T).3: i32)     
+	n[5]:  addr.load  n[4]  => _   @ bb2[9]:  fn test_ref_field; _7 = (((*_6).3: pointers::T).3: i32)     
+	n[6]:  field.3    n[2]  => _   @ bb2[10]: fn test_ref_field; (((*_6).3: pointers::T).3: i32) = move _7
+	n[7]:  field.3    n[6]  => _   @ bb2[10]: fn test_ref_field; (((*_6).3: pointers::T).3: i32) = move _7
+	n[8]:  addr.store n[7]  => _   @ bb2[10]: fn test_ref_field; (((*_6).3: pointers::T).3: i32) = move _7
+	n[9]:  copy       n[0]  => _10 @ bb2[16]: fn test_ref_field; _10 = &raw mut (*_6)                     
```

Comment:
This has source `n[0]`, but I think it should be `n[2]`, since that's the last node that assigned to `_6`.

Reply:
Fixed in 4ca53bd3